packages/in_app_purchase/in_app_purchase_android/lib/src/types/google_play_purchase_details.dart

```diff
@@ -19,31 +19,21 @@ class GooglePlayPurchaseDetails extends PurchaseDetails {
     required String? transactionDate,
     required this.billingClientPurchase,
     required PurchaseStatus status,
+    bool pendingCompletePurchase = false,
   }) : super(
-            productID: productID,
-            purchaseID: purchaseID,
-            transactionDate: transactionDate,
-            verificationData: verificationData,
-            status: status) {
-    this.status = status;
+          productID: productID,
+          purchaseID: purchaseID,
+          transactionDate: transactionDate,
+          verificationData: verificationData,
+          status: status,
+        ) {
+    this.pendingCompletePurchase = pendingCompletePurchase;
```

Comment:
Since we already have `billingClientPurchase`, can we just use it?
`pendingCompletePurchase = !billingClientPurchase.isAcknowledged`
So we don't need the extra `pendingCompletePurchase` parameter.

Reply:
You are 100% correct, thank you for this suggestion. I have committed the necessary changes.